File: HFY.Entities/Concrete/AracBakimlar.cs
using HFY.Core.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace HFY.Entities.Concrete
{
    public class AracBakimlar : IEntity
    {
        [Key]
        public int AracBakimID { get; set; }


        public int AracID { get; set; }
        public int LastikID { get; set; }
        public int AksPozisyonID { get; set; }
        public int OlusturanId { get; set; }
        public DateTime OlusturmaTarihi { get; set; }
        public int DuzenleyenId { get; set; }
        public DateTime DuzenlemeTarihi { get; set; }
        public bool Aktif { get; set; }

    }
}

File: HFY.MvcWebUI.Angular/ClientApp/src/app/lazy-general-page/classes/arac-bakimlar/arac-bakimlar.ts
export class AracBakimlar {
    AracBakimID: number;
    AracID:number;
    LastikID: number;
    AksPozisyonID: number;
    OlusturanId: number;
    OlusturmaTarihi: Date | string;
    DuzenleyenId: number;
    DuzenlemeTarihi: Date | string;
    Aktif: boolean;

    // Bu kısmın alt tanımları, araç bakım sayfasında bazı kontroller için eklenmiştir. veritabanında bu haneler bulunmamaktadır.
    DoluBos:boolean
    GeldigiYer:string; // Örn;Depo
    BulunduguYer:string;
    SeriNo:string // Aynı seri no iki kere girilemez.
    LastikTipID:number; // Lastiğin rengini bulmak için.
    AracKilometre:number // Araç kilometresini elde tutmak için.
    ServisTarihi:string; // Servis tarihini elde tutmak için.
}
